scripts/generate-flow-types.sh

```diff
@@ -0,0 +1,3 @@
+# /bin/sh
+
+for file in $(find ./lib -name *.d.ts -type f); do flowgen --add-flow-header ${file} -o ${file/.d.ts/.js.flow}; done;
```

Comment:
I would rewrite this like that:
```
#!/usr/bin/env bash

set -o errexit
set -o pipefail
set -u

readonly IFS=$'\n\t'

_main() {
  for file in $(find ./lib -name *.d.ts -type f); do
    flowgen --add-flow-header "${file}" -o "${file/.d.ts/.js.flow}"
  done;
}

_main "$@"

```

In its current (the PR) form:
- flowgen will continue to process files even if there is an error (and the error will be masked). 
- File names with space could be an issue (won't be an issue, but shellcheck does not like it).
- That's a long line.

Reply:
Sounds good! Thanks for reviewing this bash file, I was holding myself not to rewrite it in JS. 😅 

Thanks to you for leading the way to TypeScript.